projects/ui-particles-angular/src/lib/gio-monaco-editor/gio-monaco-editor.component.ts

```diff
@@ -79,6 +80,10 @@ export class GioMonacoEditorComponent implements ControlValueAccessor, AfterView
   @Input()
   public disableAutoFormat = false;
 
+  @Input()
+  @HostBinding('class.single-line')
+  public singleLineMode = false;
```

Comment:
Do you think that it would be interesting to have component inheritence for this use case. We would have this GioMonacoEditorComponent and something like GioInlineMonacoEditorComponent ?
We would this second component that have one more input and should be able to override parent methods 🤔